### PR TITLE
Release 5.3.2

### PR DESCRIPTION
**Pull request by JordanBalesBAO**
## Developer: Jordan Bales

This is the release branch candidate for datapackr 5.3.2. 

Note any fixes will need to be applied to this branch not dev.

### CODE REVIEW

**Review by jason-p-pickering — APPROVED** [review]

**Comment by JordanBalesBAO** [comment]
So I never updated the description file last time we released, but in github and the associated tags I did use 5.3.1 every where.  Im going to push to update the description again unless there is any concerns? 

I have the process documented in confluence, so maybe we meet and fill in some gaps I have?

File: R/createAnalytics.R [hunk 1]
```diff
@@ -46,53 +13,50 @@ getFY22Prioritizations <- function(d) {
...
+    d$data$analytics %<>%
+          dplyr::mutate(dplyr::across(
+            c(mechanism_code, mechanism_desc, partner_desc, funding_agency),
+            ~ dplyr::case_when(
+              is.na(.x) &
+                stringr::str_detect(support_type, "DSD|TA") ~ "Unallocated",
+              is.na(.x) &
+                stringr::str_detect(support_type, "Sub-National") ~ "default",
+              is.na(.x) &
+                stringr::str_detect(support_type, "No Support Type") ~ "default",
```
[the remark this answers is not in the document]
Comment (jacksonsj) [reply R/createAnalytics.R]: I've made this change

File: R/adorn_import_file.R [hunk 2]
```diff
@@ -65,76 +72,94 @@
...
-      )
+    psnu_import_file %<>%
+      addcols("prioritization")
+    #TODO: Rename this everywhere to something specifying it means psnu
+    # prioritization, instead of facility/community
   } else {
-    # If psnu_prioritizations are found
-    prio_defined <- prioritization_dict() %>% # Dict found in utilities.R
-      dplyr::select(value, prioritization = name)
+    # Check prioritizations
```
[the remark this answers is not in the document]
Comment (jacksonsj) [reply R/adorn_import_file.R:114]: I've made this change

**Comment by jacksonsj** [comment]
@JordanBalesBAO - Approved from me

**Review by jacksonsj — APPROVED** [review]

**Pull request merged**